vignettes/04-synthesis-data.Rmd

```diff
@@ -31,7 +31,7 @@ trait_canopy_height <- betydb_query(table     = "search",
                       trait     = "canopy_height", 
                       date      = "~2017",
                       limit     =  "none")
-trait_canopy_height = trait_canopy_height %>% 
+trait_canopy_height2 = trait_canopy_height %>% 
```

Comment:
I don't like names that are too long, but maybe trait_canopy_height_day or trait_canopy_height_by_day?